src/components/Projects/Projects.js

```diff
@@ -1,34 +1,56 @@
+import React, { useState } from "react";
+
 import ProjectItem from "./ProjectItem";
-import "./Projects.css";
 import Card from "../UI/Card";
 
+import "./Projects.css";
+
 function Projects(props) {
+  const [isChecked, setIsChecked] = useState(false);
+
+  const changeStateHandler = (event) => {
+    setIsChecked(!isChecked);return;
+  };
```

Comment:
The checkbox should be changeable for each item individually. And it should be shown on each item as well